test/runner.py

```diff
@@ -2,8 +2,6 @@
 import os
 import sys
 
-sys.path.insert(0, os.path.dirname(os.path.dirname(__file__)))
```

Comment:
are you asking me to keep this around, or giving a :+1:?

there is no installed version of the tests...